detections/endpoint/macos_system_network_configuration_discovery.yml

```diff
@@ -0,0 +1,65 @@
+name: MacOS System Network Configuration Discovery
+id: 66945806-0891-4055-952f-ea11b7bc54dd
+version: 1
+date: '2025-09-08'
+author: Jamie Windley
+status: production
+type: Anomaly
+description: This analytic detects the execution of common network configuration discovery commands on macOS systems. Specifically, it focuses on instances of `netstat -ant`, `arp -a`, and `ifconfig`. These utilities are often used for legitimate troubleshooting, but they are also frequently leveraged by adversaries during the reconnaissance phase to enumerate network interfaces, identify active connections, and discover other hosts within the same broadcast domain. Monitoring for the use of these commands, particularly when executed by unexpected users or during unusual time periods, can provide early warning of potentially malicious activity.
+data_source:
+- osquery
+search: "| tstats `security_content_summariesonly` values(Processes.process) as process\
+  \ values(Processes.parent_process) AS parent_process values(Processes.parent_process_exec)\
+  \ AS parent_process_exec values(Processes.parent_process_id) AS parent_process_id\
+  \ values(Processes.parent_process_name) AS parent_process_name values(Processes.parent_process_path)\
+  \ AS parent_process_path min(_time) as firstTime max(_time) as lastTime from datamodel=Endpoint.Processes\
+  \ where Processes.process IN (\"netstat -ant\", \"arp -a\", \"ifconfig\") by Processes.action\
+  \ Processes.dest Processes.process Processes.process_hash Processes.process_id Processes.process_name\
+  \ Processes.process_path Processes.user Processes.vendor_product \n| `drop_dm_object_name(Processes)`\
+  \ \n| `security_content_ctime(firstTime)` \n| `security_content_ctime(lastTime)`\n\
+  | `macos_system_network_configuration_discovery_filter`\n | `macos_system_network_configuration_discovery_filter`"
```

Comment:
We already have generic coverage in CIM for this in the rule https://github.com/splunk/security_content/blob/develop/detections/endpoint/linux_system_network_discovery.yml

You just need to updated the Data Source and linked test in that rule.

We could debate if its necessary updating the name (because its incorrect now that macos is added).